navigation_commands:
  "ミャクミャクの椅子へ向かう": "ミャクミャクの椅子"
  "抹茶の箱へ向かう": "抹茶の箱"
  "3Dプリンターへ向かう": "3Dプリンター"

unsupported_patterns:
  - "にあるアイテムを探す"
  - "にあるクッキーを掴む" 
  - "にある皿を掴む"
  - "にある遊び道具を掴む"
  - "でアイテムを手放す"

special_commands:
  "行動終了": "END_ACTION"